src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/Results.tsx

```diff
@@ -103,26 +100,15 @@ export const Results: React.FC<Props> = ({
               departureTime={tripPattern.expectedStartTime}
               previousDepartureTime={tripPatterns[i - 1]?.expectedStartTime}
             />
-            {newTravelSearchEnabled ? (
-              <ResultItem
-                tripPattern={tripPattern}
-                onDetailsPressed={() => {
-                  onDetailsPressed(tripPattern, i);
-                }}
-                searchTime={searchTime}
-                testID={'tripSearchSearchResult' + i}
-                resultNumber={i + 1}
-              />
-            ) : (
-              <ResultItemOld
```

Comment:
Should old components also be removed as a part of this?

Reply:
Good catch, I really thought I had deleted it. :+1: